src/components/ContentHighlights/HighlightStepper/HighlightStepperConfirmContent.jsx

```diff
@@ -43,18 +45,40 @@ const BaseReviewContentSelections = ({
   }
 
   const { hits } = camelCaseObject(searchResults);
+  const contentCards = () => {
+    const cardData = hits.map((highlightedContent) => {
+      const {
+        aggregationKey, title, cardImageUrl, contentType, partners, originalImageUrl, firstEnrollablePaidSeatPrice,
+      } = highlightedContent;
+      const original = {
+        aggregationKey,
+        title,
+        contentType,
+        partners,
+        cardImageUrl,
+        originalImageUrl,
+      };
+      original.extras = {
+        firstEnrollablePaidSeatPrice,
+      };
+      return original;
+    });
+    return cardData;
+  };
 
+  // eslint-disable-next-line react-hooks/rules-of-hooks
```

Comment:
Generally a code smell if you have to disable this ESLint rule. Hooks need to happen consistently across all renders, and this code here is after a conditional above. 

[curious] Any particular reason why you're passing `contentCards()` as React state if it never gets updated?